doc/installation/linux_install.rst

```diff
@@ -69,55 +86,26 @@ The following are the basic steps to compile and install NEST from source code:
     make install
     make installcheck
 
-NEST should now be successfully installed on your system. You should now be able to ``import nest``  from a python or ipython shell.
-
-.. admonition:: IMPORTANT!
-
- If your operating system does not find the ``nest`` executable or if Python does not find the ``nest`` module, your path variables may not be set correctly. This may also be the case if Python cannot load the ``nest`` module due to missing or incompatible libraries. In this case, please run:
-
-  .. code-block:: sh
+NEST should now be successfully installed on your system.
 
-       source </path/to/nest_install_dir>/bin/nest_vars.sh
+* Before using NEST, make sure that all the environment variables are set correctly. See the section `Environment variables`_ for details.
```

Comment:
```suggestion
* Before using NEST, make sure that all required environment variables are set correctly. See the section `Environment variables`_ for details.
```

Alternatively, the following could be a nice shortcut for people who already know their way around:

```suggestion
* Before using NEST, make sure that all required environment variables are set correctly. In short, this can be established by sourcing the shell script `nest_vars.sh`, which is installed into the path for binaries selected during the CMake run. See the section `Environment variables`_ for details.
```